app/assets/javascripts/hyrax/cable.js

```diff
@@ -0,0 +1,12 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the rails generate channel command.
+//
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
+  this.App || (this.App = {});
+
+  App.cable = ActionCable.createConsumer("/notifications/endpoint");
```

Comment:
This is hard-coded at the moment to the new path from `config/routes.rb`

Reply:
Are you trying to avoid erb'ing this file to pull a named route in, or is that TBD?

Reply:
Laziness, to see if folks cared. I'll make this change. :)